<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AchievementSettingsWidget</class>
 <widget class="QWidget" name="AchievementSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>481</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaAchievements">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-2</y>
        <width>655</width>
        <height>739</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="settingsBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QCheckBox" name="unofficialAchievements">
            <property name="text">
             <string>Test Unofficial Achievements</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="hardcoreMode">
            <property name="text">
             <string>Enable Hardcore Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="enable">
            <property name="text">
             <string>Enable Achievements</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="spectatorMode">
            <property name="text">
             <string>Enable Spectator Mode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="encoreMode">
            <property name="text">
             <string>Enable Encore Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="notificationBox">
         <property name="title">
          <string>Notifications</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="achievementNotificationsDurationSliderContainer" stretch="1,0">
            <item>
             <widget class="QSlider" name="achievementNotificationsDuration">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="achievementNotificationsDurationLabel">
              <property name="text">
               <string>5 seconds</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="achievementNotifications">
            <property name="text">
             <string>Show Achievement Notifications</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="leaderboardNotificationsDurationSliderContainer" stretch="1,0">
            <item>
             <widget class="QSlider" name="leaderboardNotificationsDuration">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TickPosition::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="leaderboardNotificationsDurationLabel">
              <property name="text">
               <string>5 seconds</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="leaderboardNotifications">
            <property name="text">
             <string>Show Leaderboard Notifications</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="soundEffects">
            <property name="text">
             <string>Enable Sound Effects</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="overlays">
            <property name="text">
             <string>Enable In-Game Overlays</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="soundEffectsBox">
         <property name="title">
          <string>Sound Effects</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="5" column="0">
           <widget class="QLineEdit" name="lbSoundPath"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="notificationSoundPath"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="unlockSoundBrowse">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="notificationSoundReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="unlockSound">
            <property name="text">
             <string>Achievement Unlock Sound</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="notificationSoundBrowse">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="unlockSoundPath"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="lbSoundBrowse">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="unlockSoundOpen">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="notificationSoundOpen">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="notificationSound">
            <property name="text">
             <string>Notification Sound</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="lbSoundOpen">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="unlockSoundReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="lbSoundReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="lbSound">
            <property name="text">
             <string>Leaderboard Submit Sound</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="loginBox">
         <property name="title">
          <string>Account</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QLabel" name="loginStatus">
            <property name="text">
             <string>Username:
Login token generated at:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="achievementButtons">
            <item>
             <widget class="QPushButton" name="viewProfile">
              <property name="text">
               <string>View Profile...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="text">
               <string>Login...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gameInfoBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>Game Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="gameInfo">
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rcheevosDisclaimer">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;PCSX2 uses RetroAchievements as an achievement database and for tracking progress. To use achievements, please sign up for an account at &lt;a href=&quot;https://retroachievements.org/&quot;&gt;retroachievements.org&lt;/a&gt;.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;To view the achievement list in-game, press the hotkey for &lt;span style=&quot; font-weight:600;&quot;&gt;Open Pause Menu&lt;/span&gt; and select &lt;span style=&quot; font-weight:600;&quot;&gt;Achievements&lt;/span&gt; from the menu.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>8</number>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
